src/fsharp/FSharp.Core.Unittests/FSharp.Core/Microsoft.FSharp.Control/AsyncType.fs

```diff
@@ -80,6 +80,15 @@ type AsyncType() =
         ()
 
     [<Test>]
+    member this.AsyncRunSynchronouslyReusesThreadPoolThread() =
+        let action = async { async { () } |> Async.RunSynchronously }
+        let computation =
+            [| for i in 1 .. 1000 -> action |]
+            |> Async.Parallel
+        Async.RunSynchronously(computation, timeout = 1000)
+        |> ignore
```

Comment:
Yes, there is no assert. The purpose of this test is to ensure that the computation completes quickly (originally it took more than 1 minute with 100 actions - now there are 1000 actions and it takes less than 1 second) - otherwise `TimeoutException` would be raised.